### PR TITLE
[chrome] Change chrome-cast's class-like interfaces to class definitions.

### PR DESCRIPTION
**Pull request by stefan-b314**
**TL;DR**: This avoids TS2339 when calling the constructor of these classes, e.g.
`new chrome.cast.SessionRequest(...)`.

**Issue Description:**

I ran into an issue when trying to use the existing chrome-cast type definitions. I wanted to create new instances of `chrome.cast.SessionRequest` and `chrome.cast.ApiConfig`, but the compiler returned with the following errors:

```
TS2339: Property 'SessionRequest' does not exist on type 'typeof cast'.
TS2339: Property 'ApiConfig' does not exist on type 'typeof cast'.
```

I figured out that this is because theses classes are currently defined as class-like interfaces and thus can not be instantiated. My changes are fixing this issue for me.

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html, https://developers.google.com/cast/docs/reference/chrome/
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

**Review by mikiest — APPROVED** [review]
This is a fairly simple change, can anyone review it as soon as possible?

I am facing the exact same problem, I identified the same cause and was about to raise a PR for it when I decided to check if one had already been created. Thanks @stefanullinger !

**Comment by couven92** [comment]
@stefanullinger Could you please fill out the first checkbox in the last group of checkboxes:

> Provide a URL to documentation or source code which provides context for the suggested changes

It would help with the review, if I didn't have to search the web for the correct documentation.

**Comment by mikiest** [comment]
@couven92 @stefanullinger This is the TypeScript documentation for defining a class in a `.d.ts` file:
https://www.typescriptlang.org/docs/handbook/declaration-files/templates/module-class-d-ts.html

**Comment by couven92** [comment]
@mikiest I was thinking of the actual Google Chromecast API documentation. I know for regular google extensions API these are located at https://developer.chrome.com/extensions/api_index

**Comment by stefanullinger** [comment]
@couven92 Sure, here you go: https://developers.google.com/cast/docs/reference/chrome/

**Review by ndillon1 — APPROVED** [review]

**Review by fredrikhr — APPROVED** [review]
:+1:
